### ISSUE TITLE
Sending SMS Problem

### ISSUE THREAD

**Issue by CL-Innovations** [jookies/jasmin#585]
I currently just installed a brand new Jasmin SMS server.
I'm sending SMS using the URL HTTP request:
http://192.168.4.153:1401/send?username=xxxxxxx&password=xxxxxxx&from=19999999999&to=18888888888&content=hello
**The usernames, passwords and phones numbers arent real, but the 1 at the begining is good.

The SMS doesnt seem to get to my SMSC SMMP server.
Could you please help me.

These are the info i think you could need tell me if you need more.

##  smppccm -l
jcli : smppccm -l
#Connector id                        Service Session          Starts Stops
#plivo_1                             started BOUND_TRX        1      0
Total connectors: 1

## mtrouter -l
jcli : mtrouter -l
#Order Type                    Rate       Connector ID(s)                                  Filter(s)
#0     DefaultRoute            0 (!)      smppc(plivo_1)
Total MT Routes: 1

## user -l
jcli : user -l
#User id          Group id         Username         Balance MT SMS Throughput
#xxxxxxxx         zzzzzzzz       xxxxxxxx         ND (!)  ND (!) ND/ND
Total Users: 1

## group -l
jcli : group -l
#Group id
#g-formatik
Total Groups: 1

## default_plivo_1.log
2017-05-29 10:24:40 INFO     9364 Reconnecting after 10 seconds ...
2017-05-29 10:25:13 INFO     1148 Establishing TCP connection to 54.67.54.200:2775
2017-05-29 10:25:13 INFO     1148 Connecting to IPv4Address(TCP, '54.67.54.200', 2775) ...
2017-05-29 10:25:13 WARNING  1148 SMPP connection established from 54.67.54.200 to port 50672
2017-05-29 10:25:13 INFO     1148 Connection made to 54.67.54.200:2775
2017-05-29 10:25:13 WARNING  1148 Requesting bind as transceiver
2017-05-29 10:25:14 WARNING  1148 Bind succeeded...now in state BOUND_TRX

## messages.log
2017-05-29 14:34:53 INFO     1148 SMS-MT [cid:plivo_1] [queue-msgid:da287db2-d8b2-410e-97e5-75c1c70d9bf2] [smpp-msgid:1bc4cf23-475c-4a2a-82f5-670d7abb218d] [status:ESME_ROK] [prio:0] [dlr:NO_SMSC_DELIVERY_RECEIPT_REQUESTED] [validity:none] [from:19999999999] [to:18888888888] [content:'hello']

**Comment by farirat** [comment]
Through your messages.log i can see the message were sent to plivo, you should check why this message were not delivered by them.

**Comment by tedfernandess** [comment]
I have the same error.

My log file:

SMS-MT [cid:DEMO_CONNECTOR] [queue-msgid:20906063-25a0-436c-b79a-921ecf613e45] [status:ERROR/RESERVEDSTATUS_VENDOR_SPECIFIC] [retry:False] [prio:0] [dlr:NO_SMSC_DELIVERY_RECEIPT_REQUESTED] [validity:none] [from:None] [to:00258842913365] [content:'ted']

What could be the problem ?

**Comment by tarikogut** [comment]
Orginator is wrong

**Comment by ZeeshanSultan** [comment]
@hzcodertr the [from:None] parameter is the originator right ?

**Comment by hzcodertr** [comment]
yes

**Comment by ArunaMahaGamage** [comment]
Let me know where is the place fix this issue

**Comment by HoLiSchitz** [comment]
I'm new to web development. I tried installing and configuring jasmine. I followed the steps from tutorial and got OK message through http request in browser for sending a message. Can someone tell me why I'm not getting an actual message to my number when I get success message? I used my system ip and some random port number for my DEMO_CONNECTOR. I don't know what to put there. It says the connector is started but when I see its status, it's not bound.
